lib/master.js

```diff
@@ -172,7 +181,7 @@ class Master extends BaseService {
                 type: 'config',
                 body: yaml.dump(config)
             });
-            const startupWorkerExit = (code) => {
+            const startupWorkerExitHandler = (code) => {
```

Comment:
We need to remove the startupWorkerExitHandler from within the startupWorkerExitHandler as well

Reply:
Same as the workerMessageHandler.

Reply:
Added both